charts/fpl-case-service/values.preview.template.yaml

```diff
@@ -10,6 +10,7 @@ java:
     CORE_CASE_DATA_API_URL: http://${SERVICE_NAME}-data-store-api
     IDAM_CLIENT_ID: fpl_case_service
     IDAM_CLIENT_REDIRECT_URI: https://fpl-case-service-*.service.core-compute-aat.internal/oauth2/callback
+    SEND_LETTER_API_URL: http://fpl-prd-mock-service
```

Comment:
do we still want this as stubbed in preview?

Reply:
I don't think so, removed